CHANGELOG.md

```diff
@@ -60,6 +60,8 @@
 
 - Applying lazy loading in About, Auth, Tools and Maintenance modules. (INDIGO Sprint 230428, [!415](https://github.com/TeskaLabs/asab-webui/pull/415))
 
+- Enable dark mode in Home screen, refactor class components to functional, update body.css in demo app, add dynamic configuration section to index.html. (INDIGO Sprint 230428, [!420](https://github.com/TeskaLabs/asab-webui/pull/420))
```

Comment:
@Samsunggora How did you enabled dark mode on HomeScreen?

Reply:
@petrKavulok not sure about that. This add a class `loaded` which is used to determine whether the app is loaded or not and display the correct background color (and loader). I think the comment above that is not understandable,cant say but I guess Ari wrote it.

What I think is that by refactoring the class component to functional Egor somehow removed the original background color, which was I guess white

Reply:
@Pe5h4  Yes, you are right. But I believe, what is meant by 'Enable dark mode in Home Screen,...' is adding this 'loaded' classname which handles correct bg color. 
I think the demo app might not have been properly refactored when we introduced the new design and the dark background color was not functional.